protocol/messenger_settings.go

```diff
@@ -20,3 +22,19 @@ func (m *Messenger) SetLogLevel(request *requests.SetLogLevel) error {
 func (m *Messenger) SetCustomNodes(request *requests.SetCustomNodes) error {
 	return nodecfg.SetWakuV2CustomNodes(m.database, request.CustomNodes)
 }
+
+func (m *Messenger) SetCustomizationColor(ctx context.Context, request *requests.SetCustomizationColor) error {
+	updatedAt := timesource.GetCurrentTimeInMillis()
+
+	acc, err := m.multiAccounts.GetAccount(request.KeyUID)
+	if err != nil {
+		return err
+	}
+	acc.CustomizationColor = request.CustomizationColor
+	acc.CustomizationColorClock = updatedAt
+	err = m.syncAccountCustomizationColor(ctx, acc)
```

Comment:
Does this save the value as well locally? (you can check if login/logout preserve the value)